src/composables/useEthers.ts

```diff
@@ -26,14 +26,22 @@ export default function useEthers() {
   const { supportsBlocknative } = useBlocknative();
   const { refetchBalances } = useTokens();
 
+  async function getTxConfirmedAt(receipt: TransactionReceipt): Promise<Date> {
+    const block = await rpcProviderService.jsonProvider.getBlock(
+      receipt.blockNumber
+    );
+    return new Date(block.timestamp * oneSecondInMs);
```

Comment:
I think it could be handy to extract it to a function - something like `toJSTimestamp`, I think I might have seen this before where we take a unix timestamp and multiply it by 1000.

Also, can `block` be null?